apps/website/src/examples/Stepper.tooltip.tsx

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Button, Flex, Stepper } from '@itwin/itwinui-react';
+
+const steps = [
+  { name: 'Completed Step', description: 'Completed Tooltip' },
+  { name: 'Current Step', description: 'Current Tooltip' },
+  { name: 'Next Step', description: 'Next Tooltip' },
+  { name: 'Last Step', description: 'Last Tooltip' },
+];
+
+export default () => {
+  const [currentStep, setCurrentStep] = React.useState(2);
```

Comment:
This is setting step 3 labeled **Next step** as the current step.
![Screenshot 2023-03-28 at 3 04 14 PM](https://user-images.githubusercontent.com/849817/228341393-8eb1a579-63ea-4d9b-a400-7098829cc5c0.png)

I think you want:
```suggestion
  const [currentStep, setCurrentStep] = React.useState(1);
```
Or you can rename the step labels.

Reply:
Good catch! This has been fixed